cvat-ui/src/components/labels-editor/constructor-viewer-item.tsx

```diff
@@ -42,7 +42,7 @@ export default function ConstructorViewerItem(props: ConstructorViewerItemProps)
                     onClick={(): void => onDelete(label)}
                     onKeyPress={(): boolean => false}
                 >
-                    <CloseOutlined />
+                    <MinusCircleOutlined />
```

Comment:
It looks quite strange, guess the close icon looked more convenient and less destructing.

Reply:
![image](https://user-images.githubusercontent.com/40690378/152488968-eb8d7403-14c2-4689-b6ae-f3a4e2b21f1f.png)

Add label: circled plus icon
Remove label: close icon

They are opposite actions, but icons not.  One circled, another not. The same for attributes.
Following from the name of CloseOutlined it is event not supposed that the icon is used to remove something

Reply:
But I undestand you feeling about distracting, changed icon attract your attention for now because you are not used to it yet

Reply:
Put it here to compare
![image](https://user-images.githubusercontent.com/40690378/152489980-5fd8dc89-b4a3-4a68-9870-6ffb46771f64.png)

Reply:
@bsekachev , which software or products use circle with minus to remove something?

Reply:
@nmanovic 

Figma:
![image](https://user-images.githubusercontent.com/40690378/152498096-0fbe91be-ecdd-483e-a111-ac75e4c33614.png)

Different options are possible: minus, trash icon, close icon. But in this specific case, I believe the best two are minus (like a pair for plus icon) and trash icon (looks well with edit icon and also very intuitive))

Reply:
For comparison: 
![image](https://user-images.githubusercontent.com/40690378/152499196-77cf294f-5ca2-4701-9a7b-670dec68e4a1.png)

Reply:
IMHO: Cross and trash bin look better. Minus in a circle looks like the stop sign. On Figma icons without circle and they are in one column. It makes them clear semantically.

Reply:
Let's use trash instead, I am ok with it.